### PR TITLE
Watch style files

### PR DESCRIPTION
**Pull request by coffee-cup**
- Watch style files
- Tests for watching style files and theme changes

Builds off #91

### CODE REVIEW

**Comment by coffee-cup** [comment]
Hmmm I keep getting this error when trying to run on this branch

image

Will debug later, but probably don't merge until this is resolved.

File: packages/server/test/watch.test.ts [hunk 1]
```diff
@@ -170,4 +176,19 @@ export const Test = () => <div />;`);
 
     watcher.close();
   });
+
+  it("does trigger callback when annotation added to css file", async () => {
```
Comment (andrejak) [inline packages/server/test/watch.test.ts]: Should we have a test for when annotation is removed too?

File: packages/server/test/watch.test.ts [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+    await wait(500);
+    await writeStyleFile("/* @snoopy:styles */");
+
+    await waitUntil(() => triggered);
+
+    watcher.close();
+  });
+
+  it("does trigger callback when annotation added to theme", async () => {
```
Comment (andrejak) [inline packages/server/test/watch.test.ts]: Same here, maybe a test that removal is picked up too?

File: packages/server/src/watch.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import {checkMatch} from "@prodo-ai/snoopy-search";
+// tslint:disable-next-line:no-submodule-imports
+import {ignored} from "@prodo-ai/snoopy-search/src/utils";
```
Comment (tdawes) [inline packages/server/src/watch.ts]: Will this work when it's built (and the code will live in `lib/` instead)? 

Why not just expose `ignored` as part of the `@prodo-ai/snoopy-search`s API?

**Pull request merged**